php/src/Core/Database/AbstractRepository.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace App\Core\Database;
+
+use App\Core\AbstractDTO;
+use App\Owner\OwnerDTO;
+use Cassandra\FutureRows;
+use Cassandra\Rows;
+
+abstract class AbstractRepository
+{
+
+    public $table = '';
+
+    public $primaryKey = '';
+
+    /**
+     * @var Connector
+     */
+    public $connection;
+
+    public function __construct()
+    {
+        $this->connection = new Connector(config('database'));

```

Comment:
I don't think this is good. 
Instead of creating a new connection each time you communicate with the database (which is very expensive), you should be using a pool of existing connections, so you can reuse them each time a repository is instantiated (but I don't really know how to do this in PHP).

Reply:
Using dependency injection, maybe... We can instantiate the Connector at Command just one time and reuse Connection to others Repositorys, here [for example](https://github.com/scylladb/care-pet/pull/119/files#diff-c63a2baf8311f49b4d16fa87bf4cae7ca0b6388c87bad8481fe28ee5b601773cR21)...

Reply:
Implemented!